frontend/src/static/js/v_summary.js

```diff
@@ -915,11 +913,10 @@ window.vSummary = {
     });
   },
   mounted() {
-    this.$store.commit('updateMergedGroup', []);
-
-    // restoring custom merged groups after watchers finish their job
+    // Delay execution of filterGroupSelection watcher
+    // to prevent clearing of merged groups
     setTimeout(() => {
-      this.restoreMergedGroups();
+      this.watcherFlag = true;
```

Comment:
Perhaps can rename this watcherFlag to be more specific `filterGroupSelectionWatcherFlag`? since only filterGroupSelection watcher uses it.